setup.py

```diff
@@ -131,20 +128,25 @@ def run(self):
             "develop": CustomDevelopCommand,
             "egg_info": CustomEggInfoCommand,
         },
-        scripts=[
-            "gandlf_run",
-            "gandlf_constructCSV",
-            "gandlf_collectStats",
-            "gandlf_patchMiner",
-            "gandlf_preprocess",
-            "gandlf_anonymizer",
-            "gandlf_verifyInstall",
-            "gandlf_configGenerator",
-            "gandlf_recoverConfig",
-            "gandlf_deploy",
-            "gandlf_optimizeModel",
-            "gandlf_generateMetrics",
-        ],
+        entry_points={
+            "console_scripts": [
```

Comment:
Thanks for the note, it might be really a point of failure in future.
I do see a following solution / idea: we define a dynamic dict of {subcommand_name: subcommand_function} that we may use everywhere in the testing. The only disadvantage I see is that if a developer adds a new command code, but forget to add it to this dict (or occasionally remove existing command from dict, or occasionally rewrite it with empty dict), we'll have all tests passed, but the commands would not be available to the user. Test util would not know it should test this command. Though it would not test new subcommand existence even now if developer forget to add it there...

What do you think?

(concerning especially `gandlf_splitCSV`: I'll add an according sub-PR)